app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -6,8 +6,7 @@ class Api::V2::ContentViewsController < Api::V2::ApiController
     before_filter :find_content_view, :except => [:index, :create, :auto_complete_search]
     before_filter :find_organization, :only => [:create]
     before_filter :find_optional_organization, :only => [:index, :auto_complete_search]
-    before_filter :load_search_service, :only => [:history, :available_puppet_modules,
-                                                  :available_puppet_module_names]
+    before_filter :load_search_service, :only => [:available_puppet_modules, :available_puppet_module_names]
```

Comment:
You can actually delete this whole line, it was missed as part of the puppet module conversion.